app/models/manageiq/providers/amazon/cloud_manager/orchestration_stack.rb

```diff
@@ -14,7 +14,7 @@ def self.raw_create_stack(orchestration_manager, stack_name, template, options =
 
   def raw_update_stack(template, options)
     options = self.class.format_v2_options(options)
-    update_options = {:template_body => template.content}.merge(options.except(:disable_rollback, :timeout))
+    update_options = {:template_body => template.content}.merge(options.except(:on_failure, :timeout_in_minutes))
```

Comment:
@bzwei What are `:disable_rollback` and `:timeout` and why are we no longer excluding them?  Where they old names that were changed and never corrected?  If so it would be good to have test coverage on that.

Reply:
They are options for creation but should be excluded for update. `:disable_rollback` is for v1, now for v2 it is `:on_failure`. `:timeout` is a typo, should be `:timeout_in_minutes`.

I tried to add a spec test but found it was hard, and the logic seems simple enough. Ln 19 will be the best place to assert the options not contain `:on_failure` and `:timeout_in_minutes`, but it is hard to reach that line which need to access a provider connection.